test/e2e/suite/conformance/certificates/acme/acme.go

```diff
@@ -80,11 +80,20 @@ func runACMEIssuerTests(eab *cmacme.ACMEExternalAccountBinding) {
 
 	(&certificates.Suite{
 		Name:                "ACME HTTP01 Issuer",
+		DomainSuffixType:    "ingress",
```

Comment:
Can these suffix types be a const somewhere?

Reply:
Changed it